### PR TITLE
fix: Reading role permissions generates 500 error backend

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

- Add field roles to get organisation users

## How did you test this code?

- Make a request /api/v1/organisations/{organisation_pk}/users/

### CODE REVIEW

File: api/users/models.py [hunk 1]
```diff
@@ -243,6 +246,10 @@ def get_user_organisation(
                 % (self.id, getattr(organisation, "id", organisation))
             )
 
+    def get_user_roles(self):
+        user_roles = UserRole.objects.filter(user=self)
+        return user_roles
```
Comment (zachaysan) [inline api/users/models.py]: If `settings.IS_RBAC_INSTALLED` is not present then this method will throw an error. Maybe catch the exception and fire a custom error to stop it from looking like a random 500 error?

File: api/users/serializers.py [hunk 2]
```diff
@@ -56,9 +60,15 @@ class Meta:
 class UserListSerializer(serializers.ModelSerializer):
     role = serializers.SerializerMethodField(read_only=True)
     join_date = serializers.SerializerMethodField(read_only=True)
+    if settings.IS_RBAC_INSTALLED:
+        roles = UserRoleSerializer(many=True, read_only=True, source="get_user_roles")
```
Comment (zachaysan) [inline api/users/serializers.py:64]: I don't really like the if-statement that leads to roles not being set on the serializer. Is this how we do it throughout the codebase? I feel like defaulting `roles` to `None` would look better.

File: api/users/models.py (hunk 1 above)
Comment (novakzaballa) [reply api/users/models.py]: Done.

File: api/users/serializers.py (hunk 2 above)
Comment (novakzaballa) [reply api/users/serializers.py:64]: I don't think this is possible since UserRoleSerializer lives in RBAC.

Comment (zachaysan) [reply api/users/serializers.py:64]: Ok makes sense.

**Review by zachaysan — APPROVED** [review]

**Pull request merged**